Makefile

```diff
@@ -40,15 +52,11 @@ elevate-cpanel: $(wildcard lib/**/*) script/elevate-cpanel.PL
 				       --leave-broken \
 				       script/$@.PL
 	mv script/$@.PL.static $@
-	MARKER="`cat maint/marker`" perl -pi -e 's|^(#!/usr/local/cpanel/3rdparty/bin/perl)|$$1\n\n$$ENV{MARKER}\n|' $@
-	VERSION=${VERSION} perl -pi -e 's/(^use constant VERSION =>) 1;/$$1 $$ENV{VERSION};/' $@
-	perltidy -b -bext="/" $@
+	MARKER="`cat maint/marker`" $(PERL_BIN)/perl -pi -e 's|^(#!/usr/local/cpanel/3rdparty/bin/perl)|$$1\n\n$$ENV{MARKER}\n|' $@
+	VERSION=${VERSION} $(PERL_BIN)/perl -pi -e 's/(^use constant VERSION =>) 1;/$$1 $$ENV{VERSION};/' $@
+	$(PERL_BIN)/perltidy -b -bext="/" $@
 	chmod +x $@
-	perl -cw elevate-cpanel
-
-build:
```

Comment:
What was previously using this target?

Reply:
That commit line looks awkward - same line is below and just pathed with the PERL_BIN